tests/acceptance/href-to-test.js

```diff
@@ -85,40 +84,18 @@ module('Acceptance | href to', function(hooks) {
     element.dispatchEvent(event);
     assert.equal(event.defaultPrevented, true, 'should prevent default');
   });
-  
+
   test('clicking an ember component with href-to should work', async function(assert) {
     await visit('/');
     await click($('#href-to-links a:contains(A component with a bound href)')[0]);
-  
+
     assert.equal(currentURL(), '/about');
   });
-  
+
   test('[BUGFIX] it works with the `click` acceptance helper', async function(assert) {
     await visit('/');
     await click($('#href-to-links a:contains(About)')[0]);
     assert.equal(currentURL(), '/about');
     assertAnchorIsActive('#link-to-links a:contains(About)', assert);
   });
-  
-  test('The event listener does\'nt leak after the app is destroyed', async function(assert) {
```

Comment:
We can't destroy the application without its destroyed state leaking into other tests under the new test framework.